### ISSUE TITLE
PCWM Transfer

### ISSUE THREAD

**Issue by jzbmw** [eclipse-tractusx/sig-release#212]
## Description

This feature encapsulates all tasks/stories necessary to change to a working model only on Tractus-x. 
This covers: 
* Changes required regarding the development workflows
* quality processes 
* testing and quality process in FOSS
* security testing
* refactor technical dependencies.

### IRS

- parent of eclipse-tractusx/item-relationship-service#222
- parent of  eclipse-tractusx/item-relationship-service#223
- parent of https://github.com/eclipse-tractusx/item-relationship-service/issues/371

### Trace-X
- parent of eclipse-tractusx/item-relationship-service#188
- parent of  parent of https://github.com/eclipse-tractusx/traceability-foss/issues/741
- parent of https://github.com/eclipse-tractusx/traceability-foss/issues/1028

## Impact

This will impact all FOSS teams, which decide to switch completely to tractus-x for development.

**Comment by mkanal** [comment]
@jzbmw 
does this cover the helmcharts (deployments) as well?

**Comment by stephanbcbauer** [comment]
Hello @jzbmw , @mkanal  

Since the feature is a 24.08 feature and the development phase is coming to an end, we need a status on the feature.

- [ ] Currently you are assigned (Responsible) → Is this correct? If not, please assign the correct contact person
- [ ] Please check whether the status (backlog, work in progress ...) is set correctly
- [ ] Please check whether there is an assignment to a committee or expert group (supported-by) to be found on the [board](https://github.com/orgs/eclipse-tractusx/projects/26/views/28)
- [ ] Please comment on the current status of the feature
- [ ] Are all SubTasks (issues from other repositories that deal with the feature) linked? → The easiest way is to mention the feature here in the issue (via the ID) so we can see which teams/repositories are involved.

If you need any clarification, please get in touch, thank you very much.
Stephan

**Comment by stephanbcbauer** [comment]
Hello @jzbmw , @mkanal 
 
In a few weeks, we want to publish Tractus-X Release 24.08. For this, we need a clear status about the features (which will be included in the release and which will be cancelled or postponed to the next release).
 
Currently, the feature is still in status `Work in progress`. Please update the status by 09.07.24 eob.
 
If the feature is not ready for release 24.08, or it is possibly not planned at all, please set the status to `Not in release`. Please remember that in this case, any dependent components must be informed.
 
Thank you and best regards.

**Comment by mkanal** [comment]
Hello @stephanbcbauer ,
Feature will  be closed as soon we got IP Clearance and former documentation is  provided to association.

**Comment by mkanal** [comment]
IP Clearance is still blocked by companies. Sending repeatedly Emails to companies currently without any update regarding approval. I am still confident and keep this open.

**Comment by stephanbcbauer** [comment]
Since this an old issue from 24.08 and I assume, this is already done, I will close it. If this is a mistake, please reopen it. Thanks
